src/test/scala/workflow/DelegatingTransformerSuite.scala

```diff
@@ -1,29 +0,0 @@
-package workflow
```

Comment:
I thought about this earlier. I've stripped this logic out of DelegatingTransformer and put it into the Pipeline apply, so it should be captured by our pipeline chaining and applying unit tests. There isn't really any unit test we can run on an individual DelegatingTransformer object anymore.